.github/workflows/ci.yml

```diff
@@ -182,7 +182,7 @@ jobs:
             # Doxygen from Ubuntu is too old, need Doxygen >= 1.10
             DOCS=OFF
           else
-            DOCS=ON
+            DOCS=OFF
```

Comment:
I'm fine with this for now. My other PR won't be so easy since it still installs Doxygen 1.13 on macOS.